database/postgres/dal/user_subscription.py

```diff
@@ -0,0 +1,39 @@
+from database.models import User, TempSubscribersData, UserClubPreferences
+
+class UserSubscriptionDAL:
+    def __init__(self, db_session):
+        self.db_session = db_session
+
+    def try_add_subscribers_data(self, preference_index):
```

Comment:
`try_...` naming suggests some response which would allow one to determine whether the "try" was successful or not.

If you want to stick to the naming - make sure the method returns something always (like None, when it failed)

Or, even better, remove the "try_" poart from naming and make sure BL catches exception if any and rethrows something we can understand (like custome exception) or just None or whatever. Because currently we have unhandled exception here if `commit` fails